core/goroutine/goRoutinesProcessor.go

```diff
@@ -0,0 +1,65 @@
+package goroutine
+
+import (
+	"bytes"
+	"strings"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+)
+
+type goRoutinesProcessor struct {
+}
+
+// NewGoRoutinesProcessor creates a new GoRoutinesProcessor
+func NewGoRoutinesProcessor() core.GoRoutinesProcessor {
+	return &goRoutinesProcessor{}
+}
+
+// ProcessGoRoutineBuffer processes a new go routine stack dump based on the previous data
+func (grp *goRoutinesProcessor) ProcessGoRoutineBuffer(
+	previousData map[string]core.GoRoutineHandlerMap, buffer *bytes.Buffer) map[string]core.GoRoutineHandlerMap {
+	allGoRoutinesString := buffer.String()
+
+	splits := strings.Split(allGoRoutinesString, "\n\n")
```

Comment:
maybe define the "\n\n" marker as a const?

Reply:
fixed